lib/project_types/script/messages/messages.rb

```diff
@@ -144,6 +144,9 @@ module Messages
 
           script_upload_cause: "Fail to upload script.",
```

Comment:
Usually, our errors refer to the command that developer ran. So instead of referring to uploading the script, we should refer to the push command. 
`Push command failed.`

Reply:
This will happen automatically with whatever command was run. The final message looks something like this: `<command failed message>. <cause - why  the command failed>. <help - what dev can do to fix>.` This specific message is the middle part, the cause.